src/lib.rs

```diff
@@ -723,6 +731,11 @@ pub enum SymbolizerFeature {
     /// By default, it is true.  However, if it is false,
```

Comment:
do you think it's a good default to enable this by default? This is very expensive feature, user might not be aware about this. I think it's better to have a safe default and disable this. It's easier for user who needs line numbers to notice that returned results don't have them and figure out that they didn't enable the option. The other way, if user didn't care about line numbers, this will go unnoticed in testing because user doesn't use line number info. And then later it will cause excessive resource usage in production.

Reply:
Agree!  I will send another PR to fix it.